### PR TITLE
TLT-3219 Qualtrics ID mapping to HUID and manually updated tracking

### PR DESCRIPTION
**Pull request by Chris-Thornton-Harvard**
This PR adds a new table, qualtrics_user, which will store a users HUID, Qualtrics ID and if they have been manually updated. 
If they have been manually updated, they will be skipped.

This also removes an unused secondary_filter method which was first created when the large Qualtrics user update was happening.

https://jira.huit.harvard.edu/browse/TLT-3219

### CODE REVIEW

File: qualtrics_link/migrations/0003_auto_20180105_1238.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        ('qualtrics_link', '0002_add_acceptance'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='QualtricsUser',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('univ_id', models.CharField(max_length=32)),
+                ('qualtrics_id', models.CharField(max_length=32)),
```
Comment (sapnamysore) [inline qualtrics_link/migrations/0003_auto_20180105_1238.py]: Is 32 sufficient for qualtrics_id? Is this diff from qualtrics username?looks like teh length of username is more than 32

File: qualtrics_link/management/commands/qualtrics.py [hunk 2]
```diff
@@ -334,93 +306,121 @@ def filter_users(slice_amount):
...
+                    # If they differ, then add to the filtered list the current persons q_id, role and division
+                    if p_division != q_division:
+                        update_dict['division'] = p_division
+                        update_person = True
+
+                    # If the Person is not a Student or Employee, or Brand Admin then they need to be updated
+                    # Or if their roles do not match
+                    if q_role not in user_type_list or p_role != q_role:
+                        update_dict['user_type'] = p_role
+                        update_person = True
```
Comment (sapnamysore) [inline qualtrics_link/management/commands/qualtrics.py]: It  it worth considering the addition  of expiration date /role_end_date check with the current discussion of role end date?  
@cmurtaugh : thoughts?

**Comment by sapnamysore** [comment]
Other than the possibility of exploring the addition of expiration date/role_end date  , looks good to me 👍

File: qualtrics_link/management/commands/qualtrics.py (hunk 2 above)
Comment (Chris-Thornton-Harvard) [reply qualtrics_link/management/commands/qualtrics.py]: Jill is creating a separate story to address this issue

File: qualtrics_link/migrations/0003_auto_20180105_1238.py (hunk 1 above)
Comment (Chris-Thornton-Harvard) [reply qualtrics_link/migrations/0003_auto_20180105_1238.py]: 32 should be sufficient. The username is different than the qualtrics ID. An example ID is: UR_7TEwrqvZr1SaXGz which is only 18 char long

**Comment by Chris-Thornton-Harvard** [comment]
Thanks for the review!

**Pull request merged**